nansat/mappers/mapper_sentinel1_l1.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, filename, gdalDataset, gdalMetadata, fast=False, fixgcp=True,
         manifest_data = self.read_manifest_data(manifest_files[0])
 
         # very fast constructor without any bands only with some metadata and geolocation
-        self._init_empty(manifest_data, self.annotation_data)
+        self._init_empty(manifest_data, self.annotation_data,filename)
```

Comment:
Can you add a space before `filename`?

Reply:
done

Reply:
I don't see it is done, so I change it myself here.
```suggestion
        self._init_empty(manifest_data, self.annotation_data, filename)
```